### ISSUE TITLE
Democratic playlist page doesn't refresh when deleting songs

### ISSUE THREAD

**Issue by stebe** [ampache/ampache#3889]
### Describe the bug

I am seeing the same issue as #3651 when updating the democratic playlist. When I delete an item from the playlist the page doesn't refresh.  I'm currently on fb1009be2579915e64b366f626314541ee600cca.
### To reproduce

1. Go to Democratic playlist page after adding some songs
2. Click on 'delete icon ![image](https://github.com/ampache/ampache/assets/4076671/9cd047a8-cef8-4bb0-9420-6e7b8fee1df3) or ![image](https://github.com/ampache/ampache/assets/4076671/dfae9818-ccd1-4313-a517-4a108918dab9)
4. Page does not automatically update although the update does show if page is manually refreshed

## Environment

* Ampache version: fb1009be2579915e64b366f626314541ee600cca

### Client type

* Web

**Comment by lachlan-00** [comment]
good catch i was about to say we did that before. must be missing the browse id as well

**Comment by lachlan-00** [comment]
try f66b7de19, added the browse id and renamed the ajax to match the action

**Comment by stebe** [comment]
Unfortunately didn't make any difference for me. Vote does get removed but page still doesn't update.

**Comment by lachlan-00** [comment]
hmm, it could be missing the extra ajax, i'll have to dig a bit more

**Comment by lachlan-00** [comment]
fixed with f55f9abf3 it wasn't sending a browse so it would make a new one and not update the current list

**Comment by lachlan-00** [comment]
the cear playlist link will also clear all the votes now with 40d30f384

**Comment by stebe** [comment]
Sorry took a while to test.  Removing a vote does now work, but as admin deleting a song does not update and when I clear  playlist all the X's become checkmarks and the vote count is zeroed. Manually refreshing the page shows that the list was actually cleared.

![Screenshot_20240327_065858](https://github.com/ampache/ampache/assets/4076671/2835c588-ca52-448d-992c-ed4331a6e297)

**Comment by lachlan-00** [comment]
that's because of how the votes are checked and removed. the objects are technically still in the browse so they don't get removed right away.

They're removed from the user_vote table and not the tmp_playlist.

the process isn't very clear though so it needs a bit more work to fully clear
